shoes-core/lib/shoes/common/state.rb

```diff
@@ -1,16 +1,30 @@
 class Shoes
   module Common
     module State
```

Comment:
I found where I did this:

https://github.com/shoes/shoes4/pull/803/files#r15267351

 Not sure I understand my reasoning then . . . what we have here seems good :+1: